styleguide/source/_patterns/01-molecules/06-components/15-table/table.twig

```diff
@@ -0,0 +1,50 @@
+<div class="table">
+
+  {% include 'atoms-h5' with { 'content': 'Lorem Ipsum Dolor Amet Consectetur Adipiscing', 'class': 'table-header_title' } %}
+
+  <table>
+    <thead>
+      <tr>
+        <th>Heading 1</th>
+        <th>Heading 2</th>
+        <th>Heading 3</th>
+        <th>Heading 4</th>
+        <th>Heading 5</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr class="odd">
```

Comment:
Suggestion to use CSS even and odd pseudo classes instead of even and odd class in html